lib/csv_export.rb

```diff
@@ -0,0 +1,70 @@
+module CsvExport
+  require 'csv'
+  require 'zip'
+
+  # generates a csv from the given model data
+  def generate_csv(data)
+    objects = data.first
+    columns = data.size == 1 ? objects.first.attributes.keys : data[1]
+
+    # we never need to export ids
+    columns.delete('id')
```

Comment:
This is good, any thoughts on also removing any password / authentication stuff if it's there? We could set up an array of columns to delete and simply call something like `PROTECTED_COLS.each { |col| columns.delete(col) }`. I don't love it since that array is coupled to the schema of our users table, but there's no specific harm in calling `Hash#delete` with a key that doesn't exist. Thoughts?